README.md

```diff
@@ -16,25 +19,56 @@
 
 # Webpack CLI
 
-Webpack CLI encapsulates all code related to CLI handling. It captures options and sends them to webpack compiler. You can also find functionality for initializing a project and migrating between versions.
+* [About](#about)
+* [Getting Started](#getting-started)
+* [How It Works](#how-it-works)
+* Commands
+  - [`webpack-cli init`](./packages/init/README.md#webpack-cli-init)
+  - [`webpack-cli add`](./packages/add/README.md#webpack-cli-add)
+  - [`webpack-cli info`](./packages/info/README.md#webpack-cli-info)
+  - [`webpack-cli migrate`](./packages/migrate/README.md#webpack-cli-migrate)
+  - [`webpack-cli remove`](./packages/remove/README.md#webpack-cli-remove)
+  - [`webpack-cli generate-plugin`](./packages/generate-plugin/README.md#webpack-cli-generate-plugin)
+  - [`webpack-cli generate-loader`](./packages/generate-loader/README.md#webpack-cli-generate-loader)
+  - [`webpack-cli serve`](./packages/serve/README.md#webpack-cli-serve)
+  - [`webpack-cli update`](./packages/update/README.md#webpack-cli-update)
+* [webpack.config.js](https://webpack.js.org/concepts/configuration/)
+* [Contributing and Internal Documentation](#contributing-and-internal-documentation)
 
-## Migration from webpack v1 to v2
+## About
 
-The `migrate` feature eases the transition from [version 1](http://webpack.github.io/docs/) to [version 2](https://gist.github.com/sokra/27b24881210b56bbaff7). `migrate` also allows users to switch to the new version of webpack without having to extensively [refactor](https://webpack.js.org/guides/migrating/).
+Webpack CLI is a CLI tool for providing a flexible set of commands for developers to increase speed when setting up a custom webpack project. As of webpack v4, webpack is not expecting a configuration file but in many cases, developers want to create a more custom webpack configuration based on their use-cases and needs. Exactly all of these cases with webpack CLI we are providing a set of tools so improve the setup of custom webpack configuration. 
 
-`webpack-cli migrate <config>`
+### How to install
 
-[Read more about migrating](MIGRATE.md)
+When you have followed the [Getting Started](https://webpack.js.org/guides/getting-started/) guide of webpack then webpack CLI is already installed!
 
-## Creating new webpack projects
+Otherwise `npm install --save-dev webpack-cli` will install it. 
 
-The `init` feature allows users to get started with webpack, fast. Through scaffolding, people can create their own configuration in order to faster initialize new projects for various of use cases.
+### Commands
 
-`webpack-cli init webpack-scaffold-<package>`
+In order to simplify many common tasks when using webpack in your projects, webpack CLI provides different commands according to their use-cases.
 
-[Read more about scaffolding](SCAFFOLDING.md)
+webpack CLI is organized as a [multi-package repository](https://github.com/lerna/lerna) and every command is developed in the `packages` Folder.
```

Comment:
`webpack` => `Webpack`

Reply:
https://webpack.js.org/branding/#the-name

Reply:
So we gotta change at the top, it was with first capital letter